server/lib/availabilitySync.ts

```diff
@@ -727,89 +867,167 @@ class AvailabilitySync {
         );
+      }
 
-        if (seasonMeta4k) {
-          seasonExistsInPlex4k = true;
-        }
+      if (!plexSeason4k) {
+        seasonExistsInPlex4k = false;
       }
     } catch (ex) {
-      if (!ex.message.includes('response code: 404')) {
-        logger.debug(`Failed to retrieve plex's children metadata`, {
+      if (ex.message.includes('404')) {
+        seasonExistsInPlex4k = false;
+      }
+
+      logger.debug(
+        `Failure retrieving season ${season.seasonNumber}, TMDB ID ${media.tmdbId}, from your 4k Plex.`,
+        {
           errorMessage: ex.message,
           label: 'AvailabilitySync',
-        });
-      }
+        }
+      );
     }
+
     // Base case if both season versions exist in plex
     if (seasonExistsInPlex && seasonExistsInPlex4k) {
-      return true;
+      logger.info(
+        `Season ${season.seasonNumber}, TMDB ID ${media.tmdbId}, exists in both Plex instances.`,
+        {
+          label: 'AvailabilitySync',
+        }
+      );
+
+      return;
     }
 
-    const existsInSonarr = await this.seasonExistsInSonarr(
+    const seasonExists = await this.seasonExistsInSonarr(
       media,
       season,
       seasonExistsInPlex,
-      seasonExistsInPlex4k
+      seasonExistsInPlex4k,
+      showExists,
+      showExists4k
     );
 
-    if (existsInSonarr) {
-      logger.warn(
-        `Season ${season.seasonNumber}, media ID ${media.id} exists in at least one Sonarr or Plex instance. Media will be updated if set to available.`,
+    if (seasonExists) {
+      logger.info(
+        `Season ${season.seasonNumber}, TMDB ID ${media.tmdbId}, exists in at least one Sonarr or Plex instance.`,
         {
           label: 'AvailabilitySync',
         }
       );
-
-      return true;
     }
-
-    return false;
   }
 
-  private async initPlexClient() {
+  private async initPlexClient(): Promise<void> {
     const userRepository = getRepository(User);
     const admin = await userRepository.findOne({
       select: { id: true, plexToken: true },
       where: { id: 1 },
     });
 
-    if (!admin) {
-      logger.warning('No admin configured. Availability sync skipped.');
-      return;
+    if (admin) {
+      this.plexClient = new PlexAPI({ plexToken: admin.plexToken });
+      await this.plexClient.getStatus();
+    } else {
+      logger.error('An admin is not configured.');
     }
+  }
 
-    this.plexClient = new PlexAPI({ plexToken: admin.plexToken });
+  private async initRadarrClient(
+    radarrServers: RadarrSettings[]
+  ): Promise<void> {
+    for (const server of radarrServers) {
+      const radarrAPI = new RadarrAPI({
+        apiKey: server.apiKey,
+        url: RadarrAPI.buildUrl(server, '/api/v3'),
+      });
+
+      if (!server.is4k) {
+        await radarrAPI.getSystemStatus();
+        this.radarrClient = radarrAPI;
+      }
+
+      if (server.is4k) {
+        await radarrAPI.getSystemStatus();
+        this.radarrClient4k = radarrAPI;
+      }
```

Comment:
Won't this override existing `radarrClient` properties if there are multiple 4k or non-4k servers?